packages/manager/apps/dedicated/package.json

```diff
@@ -144,6 +144,7 @@
     "qrcode.js": "janantala/qrcode.js#~1.0.x",
     "randexp": "fent/randexp.js#~0.4.0",
     "raphael": "^2.2.8",
+    "regenerator-runtime": "^0.13.7",
```

Comment:
Is it only required for `@ovh-ux/manager-dedicated` package? 
Because I'm having trouble to understand why it's here and not added across all applications.

Reply:
@antleblanc  I think I forgot to remove it, I'll do it while fixing conflicts :)